administrator/components/com_content/src/Model/ArticlesModel.php

```diff
@@ -199,6 +199,10 @@ protected function getListQuery()
 
         $params = ComponentHelper::getParams('com_content');
 
+        // The query optimizer of both, mariadb and mysql fails to recognize the correct index on large article sets
+        // For these cases, we enforce the usage of the indexes
+        $primaryIndexAppendix = $db->getServerType() === 'mysql' ? " FORCE INDEX(PRIMARY)" : "";
```

Comment:
```suggestion
        $primaryIndexAppendix = $db->getServerType() === 'mysql' ? ' FORCE INDEX(PRIMARY)' : '';
```
PHP CS: Single quotes whenever possible.